Sources/Scheduler.swift

```diff
@@ -19,6 +19,9 @@ public protocol SchedulerProtocol {
 	///
 	/// When the work is executed depends on the scheduler in use.
 	///
+	/// - parameters:
+	///   - action: Action to schedule.
```

Comment:
This could use an article, and perhaps should it be in passive voice?
s/Action/The action/.